manual_jb/content/pyincore/install_pyincore.md

```diff
@@ -22,6 +22,11 @@ These steps guides you on how to install both pyIncore and Jupyter Notebooks on
    Version information for pyincore and pyincore-viz can be found in
    - https://anaconda.org/IN-CORE/pyincore
    
+   When installing with Jupyter, there may be an error related to Fiona package. 
```

Comment:
I might suggest a slight rewording of this. Something like:

**Note** When installing with Jupyter, there may be an error related to Fiona package.
   It is commonly caused by fiona being downgraded when installing jupyter.
   In that case, try to install jupyter first then install pyincore.
   Otherwise, you can upgrade fiona by using `conda update fiona` command.

Reply:
If you prefer, I can push the change myself.